src/solver/application.cpp

```diff
@@ -234,6 +235,16 @@ void Application::prepare(int argc, char* argv[])
     // Loading the study
     readDataForTheStudy(options);
 
+    // Collecting info from the study for a further record
+    Benchmarking::StudyInfo study_info(*pStudy);
+    study_info.collect();
+
+    // Flush previous info into a record file
+    Yuni::String filePath; 
+    filePath.clear() << pStudy->folderOutput << Yuni::IO::Separator << "study-info.txt";;
+    study_info.flush(filePath);
```

Comment:
I suggest splitting responsabilities :

- 1 class stores data
- Multiple classes can write data in CSV, human-readable format, JSON, etc.
- Possibly one interface class for those writers